multimodal/src/autogluon/multimodal/configs/matcher/default.yaml

```diff
@@ -1,21 +1,15 @@
 matcher:
-  match_label: 0
+  loss:
+    type: "contrastive_loss"
+    pos_margin: 0.8
+    neg_margin: 0.2
+  distance:
+    type: "cosine_similarity"
+  miner:
```

Comment:
We can later add some documentations about “miner”. May be via a paper/tech blog.

Reply:
Agree. We will introduce the usage in the `Customize AutoMM` tutorial. We can refer to here: https://kevinmusgrave.github.io/pytorch-metric-learning/miners/#miners